coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -190,16 +187,14 @@ private[filodb] final class NodeCoordinatorActor(metaStore: MetaStore,
 
   def coordinatorReceive: Receive = LoggingReceive {
     case e: CoordinatorRegistered     => registered(e)
-    case e: ShardIngestionState       => forward(e, e.ref, sender())
+    case s: ShardIngestionState       => logger.debug(s"Received IngestionState/Snapshot ${s.map}")
+                                         shardMaps.put(s.ref, s.map)
+                                         forward(s, s.ref, sender())
     case Terminated(memstoreCoord)    => terminated(memstoreCoord)
     case MiscCommands.GetClusterActor => sender() ! clusterActor
     case StatusActor.GetCurrentEvents => statusActor.foreach(_.tell(StatusActor.GetCurrentEvents, sender()))
     case ClearState(ref)              => clearState(ref)
     case NodeProtocol.ResetState      => reset(sender())
-    case CurrentShardSnapshot(ds, mapper) =>
```

Comment:
Is something still sending this message out? Should it be removed as well?

Reply:
The message is still being used to return current status in NodeClusterActor and is used in a ton of tests.